api.raml

```diff
@@ -273,6 +273,22 @@ schemas:
     responses:
       200:
         description: The new user record was created.
+        body:
+          application/json:
+            description: The newly-created user record.
+            schema: User
+            example: |
+              {
+                "employee_id": "12345",
+                "first_name": "John",
+                "last_name": "Smith",
+                "display_name": "John Smith",
```

Comment:
I guess it seems better to have the ID Broker generate the display name (when it's requested and the field is `null`) rather than having each app that gets info from the ID Broker having to separately duplicate that logic of how to generate a display name when not given one.